subscription/README.md

```diff
@@ -0,0 +1,12 @@
+# subscription
+
+This project contains the subscription-worker python tool. This tool reads messages from an AWS SQS queue, checks to see if the user has permission to view these records, and then publishes them to the cmr-subscription-{env} topic, so that external users can receive messages to their subscriptions.
+
+## To build and deploy this project manually
```

Comment:
can you mention this is to build and deploy project manually in AWS (not local)

Reply:
I have added the text.